scipy/stats/_discrete_distns.py

```diff
@@ -514,6 +526,124 @@ def _logcdf(self, k, M, n, N):
 hypergeom = hypergeom_gen(name='hypergeom')
 
 
+class nhypergeom_gen(rv_discrete):
+    r"""A negative hypergeometric discrete random variable.
+
+    The negative hypergeometric distribution models the probability
+    of :math:`k` successes in a sample of exactly :math:`r` failures
+    taken without replacement where :math:`N` are the total number
+    of elements and :math:`K` are defined to be successes.
+
+    It is equivalent to observing :math:`k` successes in :math:`k+r-1`
```

Comment:
should we move the relationshipto hypergeom to the notes? maybe an example would help (I did not understand it)

Reply:
I have shifted the relationship to `Notes` section and given an example in the `Examples` section. Is it ok?